app/api/items/update_item.feature

```diff
@@ -0,0 +1,186 @@
+      """
+      {
+        "type": "Course",
+        "url": "http://myurl.com/",
+        "text_id": "Task number 1",
+        "title_bar_visible": true,
+        "custom_chapter": true,
+        "display_details_in_parent": true,
+        "uses_api": true,
+        "read_only": true,
+        "full_screen": "forceYes",
+        "show_difficulty": true,
+        "show_source": true,
+        "hints_allowed": true,
+        "fixed_ranks": true,
+        "validation_type": "AllButOne",
+        "validation_min": 1234,
+        "unlocked_item_ids": "12,34",
+        "score_min_unlock": 34,
+        "team_mode": "All",
+        "teams_editable": true,
+        "team_in_group_id": "12345",
+        "team_max_members": 2345,
+        "has_attempts": true,
+        "access_open_date": "2018-01-02T03:04:05Z",
+        "duration": "01:02:03",
+        "end_contest_date": "2019-02-03T04:05:06Z",
+        "show_user_infos": true,
+        "contest_phase": "Analysis",
+        "level": 345,
+        "no_score": true,
+        "group_code_enter": true,
+        "default_language_id": "3",
+        "children": [
+          {"item_id": "12", "order": 0},
+          {"item_id": "34", "order": 1}
+        ]
+      }
+      """
+    Then the response should be "updated"
+    And the table "items" at ID "50" should be:
+      | ID | sType  | sUrl              | idDefaultLanguage | bTeamsEditable | bNoScore | sTextID       | bTitleBarVisible | bCustomChapter | bDisplayDetailsInParent | bUsesAPI | bReadOnly | sFullScreen | bShowDifficulty | bShowSource | bHintsAllowed | bFixedRanks | sValidationType | iValidationMin | idItemUnlocked | iScoreMinUnlock | sTeamMode | bTeamsEditable | idTeamInGroup | iTeamMaxMembers | bHasAttempts | sAccessOpenDate      | sDuration | sEndContestDate      | bShowUserInfos | sContestPhase | iLevel | bNoScore | groupCodeEnter |
+      | 50 | Course | http://myurl.com/ |                 3 |              1 |        1 | Task number 1 | 1                | 1              | 1                       | 1        | 1         | forceYes    | 1               | 1           | 1             | 1           | AllButOne       | 1234           | 12,34          | 34              | All       | 1              | 12345         | 2345            | 1            | 2018-01-02T03:04:05Z | 01:02:03  | 2019-02-03T04:05:06Z | 1              | Analysis      | 345    | 1        | 1              |
+    And the table "items_strings" should stay unchanged
+    And the table "items_items" should be:
+      | idItemParent | idItemChild |
+      | 21           | 60          |
+      | 50           | 12          |
+      | 50           | 34          |
+    And the table "items_ancestors" should be:
+      | idItemAncestor | idItemChild |
+      | 21             | 60          |
+      | 50             | 12          |
+      | 50             | 34          |
+    And the table "groups_items" should stay unchanged
+
+  Scenario: Valid with empty full_screen
```

Comment:
I don't get in what it is an interesting scenario. Probably there is but I don't see it.

Reply:
It is interesting because the sFullScreen column is defined as
```
`sFullScreen` enum('forceYes','forceNo','default','') NOT NULL DEFAULT 'default'
```
So an empty string should be valid.